xbmc/GUIInfoManager.cpp

```diff
@@ -5871,6 +5871,28 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     @skinning_v19 **[New Infolabel]** \link ListItem_UniqueID `ListItem.UniqueID(name)`\endlink
 ///     <p>
 ///   }
+///   \table_row3{   <b>`ListItem.BitRate`</b>,
+///                  \anchor ListItem_BitRate
+///                  _string_,
+///     @return The bitrate of a song. Actual rate for CBR, average rate for VBR.
+///     <p><hr>
+///     @skinning_v19 **[New Infolabel]** \link ListItem_BitRate `ListItem.BitRate`\endlink
+///     <p>
+///   }///   \table_row3{   <b>`ListItem.SampleRate`</b>,
+///                  \anchor ListItem_SampleRate
+///                  _string_,
+///     @return The sample rate of a song eg 44100, 48000, 96000 etc.
```

Comment:
This probably needs a adjustment since what the info label returns is 44.1 etc. 
Sorry for such a tedious comment

Reply:
No need to be sorry, you're quite correct.  Comment updated :)